src/load_csv_data.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+var extend = require('lodash').assign;
+var mysql = require('mysql');
+var config = require('../config');
+
+
+function getConnection () {
```

Comment:
We're using dependency injection, so rather than creating the connection here it should be passed into this file instead. See the way its being done through the api to the DAOs